consensus/istanbul/backend/announce.go

```diff
@@ -302,83 +390,92 @@ func (sb *Backend) handleIstAnnounce(payload []byte) error {
 	}
 	logger.Trace("Handling an IstanbulAnnounce message", "from", msg.Address)
 
-	// If the message is originally from this node, then ignore it
-	if msg.Address == sb.Address() {
-		logger.Trace("Received an IstanbulAnnounce message originating from this node. Ignoring it.")
-		return nil
-	}
```

Comment:
This is gone because the caching take over that function, or no?

Reply:
Yes, that is correct.